src/views/Notification/index.js

```diff
@@ -80,12 +80,12 @@ class Notifications extends React.Component {
         return (
             <React.Fragment>
                 { flashMsgSpan &&
-                    <div className='notification'
+                    <dialog className='notification'
```

Comment:
The change from div to dialog changes the banners look, 
![image](https://user-images.githubusercontent.com/50320115/203259671-2c3c0236-d55f-4005-b844-410ce71ce946.png), 
the default dialog padding/border should be set to none/0 so that the banner looks the same as before. 
![image](https://user-images.githubusercontent.com/50320115/203260145-1185e2a7-9fb0-4023-b07e-425ade59f02d.png)

Reply:
changed the styling only for the notification banner because we didn't want to change the styling of all dialog elements.
![notification-default-padding-border](https://user-images.githubusercontent.com/117433931/203266540-8e0f5ae9-07d3-4bdc-86da-d702461eefef.png)